the-blue-alliance-ios/Core Data/Event/Event.swift

```diff
@@ -402,7 +402,7 @@ extension Event: Locatable, Surfable, Managed {
         }
     }
 
-    public var weekString: String {
+    public var title: String {
```

Comment:
`title` on an Event seems like it would be the name of an event. We might consider naming this better. Maybe something like, `eventGroupString` or `eventGroupingString`?